examples/RLHF/ppo_trainer.py

```diff
@@ -415,7 +415,7 @@ def full_training_step(self: Trainer, inputs: Dict[str, paddle.Tensor], **kwargs
             self.scaler.step(self.optimizer)
             self.scaler.update()
             scale_after = self.scaler._scale
-            optimizer_was_run = not self.scaler._cache_founf_inf
+            optimizer_was_run = not self.scaler._cache_found_inf
```

Comment:
Please check if there's a way to ensure compatibility between interfaces. We've encountered an issue where using PaddleNLP's development version with PaddlePaddle v2.6.0 results in incompatibility.

Reply:
![image](https://github.com/PaddlePaddle/PaddleNLP/assets/4617245/94c7bbab-99b2-47c7-9431-2ee2733d2604)
Changed to check _cache_founf_inf